src/SALib/analyze/morris.py

```diff
@@ -280,9 +311,10 @@ def _reorganize_output_matrix(
     increase: bool
         Direction to consider (values that increased or decreased). "Increase"
         is the default value.
+
     Returns
     -------
-
+    numpy.ndarray
```

Comment:
```suggestion
    np.ndarray
```

Reply:
Actually, @tupui what's the norm for these? Should we be specifying the package name fully?